src/lib/Libwin/pid.c

```diff
@@ -129,7 +128,7 @@ printpids(void)
 	char	logb[LOG_BUF_SIZE] = {'\0' } ;
 
 	for (i=0; i < pids_cnt; i++) {
-		sprintf(logb, "printpids: pid_handles[%d] = %p", i, pid_handles[i]);
+		sprintf(logb, "printpids: pid_handles[%d] = %d", i, pid_handles[i]);
```

Comment:
Can we use log_buffer here? We can remove logb variable. 
Also I would suggest renaming [logb](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/logb-logbf-logbl-logb-logbf?view=vs-2019) to something else.

Reply:
Thank You @npadole20 I have removed all the local log buffers from entire file. see https://github.com/PBSPro/pbspro/pull/1765/commits/346f8d488e930ae0403dae9597bc32d7deec9823